core/stream/curve.go

```diff
@@ -37,7 +37,7 @@ func (m *mapping) transform(count int, f func(float64) float64) error {
 			stride: 64,
 		},
 	}
-	if err := tmp.conv(count); err != nil {
+	if err := tmp.conv(count, 0, 0); err != nil {
```

Comment:
Is it okay to pass 0 as min and max? Does this function expect a ftou conversion?

Reply:
`tmp` is a float format, so i think this is fine.